test/e2e/common/node/pod_resize.go

```diff
@@ -60,18 +60,18 @@ func doPodResizeTests(f *framework.Framework) {
 			containers: []e2epod.ResizableContainerInfo{
 				{
 					Name:      "c1",
-					Resources: &e2epod.ContainerResources{CPUReq: "100m", CPULim: "100m", MemReq: "200Mi", MemLim: "200Mi"},
+					Resources: &e2epod.ContainerResources{CPUReq: "10m", CPULim: "10m", MemReq: "20Mi", MemLim: "20Mi"},
```

Comment:
As I understand it, this is just the resource request and limit for the pod, not the actual usage, so it doesn't make sense to change this, right?

Reply:
Scheduler/kubelet does look at resource requests before deciding to schedule/run the pod. In other words, total pod resource requests won't exceed node resource requests even though the actual resource usage is low. 

Due to this constraint, we cannot run tests in parallel.

Reply:
I see. I ran these tests in parallel multiple times and doesn't seem to cause any issues so far. 

These tests take 1.5-2 hours if run serially and 10 mins if run with parallelism.

Reply:
Thanks for this change. They can (and should) be run in parallel as long as they don't step on the scheduler-focussed tests. Can you please verify that the worst case memory usage profile for the containers never exceed the reduced memory limits where applicable?